app/jobs/custom_filter_records_count_update_job.rb

```diff
@@ -0,0 +1,14 @@
+class CustomFilterRecordsCountUpdateJob < ApplicationJob
+  queue_as :low
```

Comment:
@sojan-official We wouldn't need this anymore; this line would take care of that, https://github.com/chatwoot/chatwoot/pull/7164/files#diff-8211fe4df2fdd38b2e0e66edd5f4b6bddc99907b9e5da03aeb006e4bb05b7a25R28

And this would happen only once until the filtered conversation count is not set in Redis, else it would fetch the count from Redis.

Let me know if this is okay then I will remove this job.

Reply:
@tejaswinichile, let's decouple the logic into two jobs. 
1. iterate over all the custom filters and  schedule sync jobs for each filter
2. sync job for a custom filter that calculates its value. 

Doing this in one single job would create a super long-running job.